package.json

```diff
@@ -35,6 +35,7 @@
     "lodash": "4.6.1",
     "mkdirp": "0.5.1",
     "moment": "2.12.0",
+    "path-is-inside": "1.0.1",
```

Comment:
I think ideally we should simply disallow what is not valid characters for the filesystem. `/^[^\/\\<>:"|?*]+$/`. This should cover at least Linux and Windows, and probably Mac as well. I think Windows is alone having such restricted file names.

Source: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx